composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "php-tuf/drupal-project",
-    "description": "Project template for Drupal projects with a relocated document root",
+    "description": "⚠️Warning: Intended for testing drupal.org TUF staging setup only. Project template for Drupal projects with a relocated document root.",
```

Comment:
We should be clear this is for testing, though we already do on the README doesn't hurt to have it here as well